Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,79 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameManager : MonoBehaviour
+{
+    private float _currentTime = 0;
+    [SerializeField]
+    private float startingTime = 60f;
+    [SerializeField]
```

Comment:
preference for putting new lines between a variable and the next attribute

```suggestion

    [SerializeField]
```